src/main/java/org/influxdb/dto/Point.java

```diff
@@ -131,7 +135,9 @@ public Builder tag(final String tagName, final String value) {
      */
     public Builder tag(final Map<String, String> tagsToAdd) {
```

Comment:
Why do we have different checks in these two tag methods ?

Reply:
That was a mistake.